docs/en/tutorials/customize_dataset.md

```diff
@@ -43,7 +43,7 @@ The annotation json files in COCO format has the following necessary keys:
  ]
 ```
 
-There are three necessary keys in the json file:
+There are three necessary keys in the JSON file:
 
 - `images`: contains a list of images with their information like `file_name`, `height`, `width`, and `id`.
 - `annotations`: contains the list of instance annotations.
```

Comment:
How about adding the necessary keys and their meanings, and adding the valid rules, such as the segmentation should have more than 6 parameters and each means x1, y1, x2, y2, ...,